data/tests/data-macro-explanation.js

```diff
@@ -0,0 +1,46 @@
+const dataMacros = /^(?:compat|css_ref|cssanimatedproperties|cssinfo|csssyntax|webextbrowsercompat)$/i;

```

Comment:
In the implementation notes maybe also say that each data macro usage requires a hidden note as opposed to one note for several data macro uses.